src/transformers/models/rwkv/modeling_rwkv.py

```diff
@@ -188,17 +188,14 @@ def backward(ctx, g_output):
             g_key,
             g_value,
         )
-        g_time_decay = torch.sum(g_time_decay, dim=0)
-        g_time_first = torch.sum(g_time_first, dim=0)
 
         return (
-            None,
-            None,
-            None,
```

Comment:
also just out of curiosity why the number of outputs changed ? 🤔

Reply:
Because autograd wants one gradient per input of the forward :-)
It used to have three ints then the variables, but no state and no bool, this PR adapts it to the changes I made.